Ensure all plugin settings are SEO_* prefixed

For example, the following settings should probably be prefixed with `SEO_[…]` in order to prevent potential collisions with other plugins.

```python
ARTICLES_LIMIT = 10
PAGES_LIMIT = 10
```

Good idea, currently there is only this two settings to update.
I didn't create the PR yet because the linter action raise an error:
```
Run invoke lint
ERROR: /home/runner/work/seo/seo/pelican/plugins/seo/seo.py Imports are incorrectly sorted.
##[error]Process completed with exit code 1.
```

But if i do it locally, everything seems good:
```
invoke lint
All done! ✨ 🍰 ✨
28 files would be left unchanged.
```

Any idea of what I'm missing?

--- a/README.md
+++ b/README.md
@@ -69,8 +69,8 @@ Example :
 You can set up a limit for article and page analysis in the plugin settings.py. By default, it's set up to 10 articles and 10 pages.
 
 ```
-ARTICLES_LIMIT = 10
-PAGES_LIMIT = 10
+SEO_ARTICLES_LIMIT = 10
+SEO_PAGES_LIMIT = 10
 ```
 
 The analysis works from the most recents articles or pages to the olders according to the `date` metadata.

--- a/pelican/plugins/seo/seo.py
+++ b/pelican/plugins/seo/seo.py
@@ -20,7 +20,7 @@ from pelican.generators import ArticlesGenerator, PagesGenerator
 
 from .seo_enhancer import SEOEnhancer
 from .seo_report import SEOReport
-from .settings import ARTICLES_LIMIT, PAGES_LIMIT, SEO_ENHANCER, SEO_REPORT
+from .settings import SEO_ARTICLES_LIMIT, SEO_ENHANCER, SEO_PAGES_LIMIT, SEO_REPORT
 
 logger = logging.getLogger(__name__)
 
@@ -49,7 +49,7 @@ def run_seo_report(generators):
 
         if isinstance(generator, ArticlesGenerator):
             # Launch analysis for each article. User can limit this number.
-            for _, article in zip(range(ARTICLES_LIMIT), generator.articles):
+            for _, article in zip(range(SEO_ARTICLES_LIMIT), generator.articles):
                 analysis = seo_report.launch_analysis(document=article)
                 documents_analysis.append(analysis)
 
@@ -58,7 +58,7 @@ def run_seo_report(generators):
 
         if isinstance(generator, PagesGenerator):
             # Launch analysis each page. User can limit this number.
-            for _, page in zip(range(PAGES_LIMIT), generator.pages):
+            for _, page in zip(range(SEO_PAGES_LIMIT), generator.pages):
                 analysis = seo_report.launch_analysis(document=page)
                 documents_analysis.append(analysis)
 

--- a/pelican/plugins/seo/settings.py
+++ b/pelican/plugins/seo/settings.py
@@ -5,5 +5,5 @@ Plugin settings : activate or deactivate SEO features
 SEO_REPORT = True
 SEO_ENHANCER = True
 
-ARTICLES_LIMIT = 10
-PAGES_LIMIT = 10
+SEO_ARTICLES_LIMIT = 10
+SEO_PAGES_LIMIT = 10

